manual/cypher/cypher-docs/src/docs/graphgists/intro/data-structures.adoc

```diff
@@ -1,16 +1,43 @@
 = Utilizing Data Structures
 
+//file:movies.csv
+//file:roles.csv
+//file:persons.csv
+//file:movie_actor_roles.csv
+
 Cypher can create and consume more complex data structures out of the box.
 As already mentioned you can create literal lists (`[1,2,3]`) and maps (`{name: value}`) within a statement.
 
-There is a number of functions that work with lists, from simple ones like `length(list)` that returns the size of a list to 
+There is a number of functions that work with lists.
+They range from simple ones like `size(list)` that returns the size of a list to `reduce`, which runs an expression against the elements and accumulates the results.
 
-// missing content here
+Let's first load a bit of data into the graph.
+If you want more details on how the data is loaded, see <<cypher-intro-importing-csv>>.
```

Comment:
Perhaps we want this helpful comment above in `compose-statements.adoc` on line 4?